addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -28,7 +28,8 @@ def _compute_show_enrich_button(self):
 
     @api.model
     def _iap_enrich_leads_cron(self):
-        timeDelta = fields.datetime.now() - datetime.timedelta(hours=1)
+        ENRICH_DELAY = float(self.env['ir.config_parameter'].sudo().get_param('crm.enrich.delay', default=1))
```

Comment:
Not sure where the param comes from but I suppose it comes from the prod

Reply:
This line introduce a new parameter. You'll found those pattern at some other place in odoo code. The idea is to give the opportunity to change the value with a config parameter but if the parameter is not define, we keep the default value, that's why there is no data for the parameter